bin/cml-runner.js

```diff
@@ -287,6 +308,12 @@ const opts = decamelize(
     .boolean('single')
     .default('single', RUNNER_SINGLE)
     .describe('single', 'If specified, exit after running a single job.')
+    .boolean('reuse-existing')
+    .default('reuse-existing', RUNNER_REUSE_EXISTING)
+    .describe(
+      'reuse-existing',
```

Comment:
```reuse-existing``` I would make it shorter and also there is one implication with the possibility of launching several with a tf file that we have not even thought.